code/datums/components/art.dm

```diff
@@ -49,3 +49,37 @@
 		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "artgreat", /datum/mood_event/artgreat)
 	else
 		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "artbad", /datum/mood_event/artbad)
+
+// NT/syndicate themed art, currently used in posters
+
+/datum/component/art/nt
```

Comment:
You should not be making a subtype, work in any additional behavior into the base art component. In this case though this behavior doesn't look like it makes sense as part of this component at all and should be part of the poster code or a second component.

Reply:
I'm just doing what the karl marx bust did

Reply:
I don't care what other code is doing, this is the wrong way to go about it

Reply:
seems like a lot of effort for components that will be used once or twice in their life

Reply:
Dealing with these changes in the proper way helps the code in the long run stay maintainable. Snowflake components subtypes really hurts the whole basis for their existence.

Reply:
I don't have a preference as to how it's done although I would have to figure out how, I just don't realize why this was not brought up at that point

Reply:
(also testing is going to be awkward until leg day is fixed)